.travis.yml

```diff
@@ -10,8 +10,8 @@ before_install:
   - travis_retry npm install --loglevel=http
   - >
     openssl aes-256-cbc \
-      -K $encrypted_4fb8e306b739_key \
-      -iv $encrypted_4fb8e306b739_iv \
```

Comment:
earlier we kept the original keys (used in your travis.yml) and existing config.json.enc, but we were getting the same result. (build failed), then we used our variables and setup a travis for the account and it worked fine. So, we kept our variables. Travis-ci has mentioned the cause of this mismatch as Security restriction : https://docs.travis-ci.com/user/pull-requests#Security-Restrictions-when-testing-Pull-Requests